<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Analysis_Dashboard</class>
 <widget class="QMainWindow" name="Analysis_Dashboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>416</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Analysis Dashboard</string>
  </property>
  <widget class="QWidget" name="centralwidget">
    <layout class="QVBoxLayout" name="main_layout">
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
     <item>
      <widget class="QWidget" name="toolbox_container" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="properties_container" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>200</height>
        </size>
       </property>
      </widget>
     </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>416</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionReset_Layout"/>
   </widget>
   <widget class="QMenu" name="menuPlots">
    <property name="title">
     <string>Plots</string>
    </property>
    <addaction name="actionClear_All_Plots"/>
   </widget>
   <addaction name="menuView"/>
   <addaction name="menuPlots"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionReset_Layout">
   <property name="text">
    <string>Reset Layout</string>
   </property>
  </action>
  <action name="actionClear_All_Plots">
   <property name="text">
    <string>Clear All Plots</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
